### DISCUSSION TITLE
Exclusive to Github ?

### DISCUSSION THREAD

**Discussion by MalikRumi** [git-lfs/git-lfs#5843] [General]
Can this tool be used with non-github repos, like gitlab, bitbucket, or even private enterprise repos? I noticed a whole subfolder dedicated to github, but not these other hosts. Thx

**Comment by chrisd8088** [comment]
Yes, absolutely, the Git LFS client is agnostic as to where your Git LFS objects are hosted.  You can host them locally, or in a variety of hosting services including GitLab, GitHub, Bitbucket, and others.  See the list of [implementations](https://github.com/git-lfs/git-lfs/wiki/Implementations) in the wiki for a longer list.
